main.tf

```diff
@@ -49,11 +49,16 @@ module "vpc_endpoints" {
   }
 }
 
+resource "random_id" "unique_prefix" {
+  byte_length = 4
+}
+
 module "runner-instance" {
   source  = "cattle-ops/gitlab-runner/aws"
   version = "7.5.0"
 
-  environment = var.environment
+  environment       = var.environment
+  iam_object_prefix = random_id.unique_prefix.hex
```

Comment:
Are you at all concerned about the final string length here?  I don't recall off-hand if/what it is for IAM names.  Would you consider adding a precondition check to make sure it's smaller than the max length?

I checked and I don't see any validation being done in `cattle-ops/gitlab-runner/aws` for this value.

Reply:
Right. It should be done. Would you mind filing an issue in the repo and it will be taken care of in a following PR?